### PR TITLE
REMOVED: duplicate Components

### PR DESCRIPTION
**Pull request by joris-rotteveel**
Removed duplicate Components after the merge. Also applied a few Code Climate suggestions.

### CODE REVIEW

File: app/assets/stylesheets/components.scss [hunk 1]
```diff
@@ -107,11 +107,11 @@ header.main {
     padding-left: 20px;
     padding-right: 20px;
     h3 {
-      color: #3593B4;
+      color: #3593b4;
```
[the remark this answers is not in the document]
Comment (dlouise64) [reply app/assets/stylesheets/components.scss]: @joris-rotteveel we should create a variables.scss for all our colours

**Comment by dlouise64** [comment]
@joris-rotteveel is this PR just for linting fixes?

**Comment by joris-rotteveel** [comment]
> @joris-rotteveel is this PR just for linting fixes?

mainly, but there are some cleanup fixes in there too

**Comment by dlouise64** [comment]
@joris-rotteveel just these things left https://codeclimate.com/github/ServiceInnovationLab/PresenceChecker/pull/141

**Pull request merged**